pages/views/NavBar.jsx

```diff
@@ -58,6 +58,10 @@ const items = [
     label: <Link href="/cefi">Cefi</Link>,
     key: "cefi",
   },
+  {
+    label: <Link href="/installment">Installment</Link>,
+    key: "installment"
+  }
```

Comment:
cool, you learned how to update the navbar by yourself 🎉

Reply:
btw, would you place it under the `Analytics` dropdown menu?

Reply:
Maybe place it under the `Analytics` dropdown menu is better. Can you suggest a better title than 'Installment' for this page?

Reply:
ChatGPT is way better than us for doing this lol

Here's an example, 
![image](https://github.com/all-weather-protocol/all-weather-frontend/assets/9366404/7c1d2d55-e221-4212-9499-9b80b5a6a29b)

It's your call since you built this page~

Reply:
`SmartInstallmentSaver` is great, sounds smarter! But this word is too long. Maybe we can keep 'Installment' title this time. When we find another word, we changed it!

Reply:
sure, it's up to you~ since you built this page haha
`Installment` also works for me!